Source/Orts.Simulation/Simulation/AIs/AI.cs

```diff
@@ -927,11 +927,20 @@ public AITrain CreateAITrainDetail(Service_Definition sd, Traffic_Service_Defini
                         }
 
                         if (train.InitialSpeed != 0)
+                        {
                             car.SignalEvent(PowerSupplyEvent.RaisePantograph, 1);
```

Comment:
I assume it is no more required to raise the pantograph, since that is handled by QuickPowerOn?

Reply:
Probably not, though technically this runs for every single car while SetPower only works on locomotives. I'll check if it's even possible to have an edge case where a non-locomotive can raise a pantograph.

Reply:
Sorry for the delay, I was stuck working on some engine file stuff rather than code. Looks to me like there probably won't be unintended consequences by removing the raise pantograph event so I've updated that for more testing.
A locomotive's pantographs are defined in MSTSWagon.cs but most useful pantograph behavior seems to be exclusively contained in MSTSElectricLocomotive.cs or assumes the pantograph is attached to a locomotive.

Reply:
Now that I think about it, it is common for EMUs to have pantographs in unpowered cars (i.e. in .wag files), so in principle this could cause those to not be raised. However, the PowerSupplyScript of the lead locomotive orders that pantographs are raised all along the train, so anyway they will be raised and it is correct to remove it as you did.

Having the RaisePantograph handled by the PowerSupply allows more flexibility over which pantographs are raised (e.g. in high speed trains usually only the pantograph on the rear locomotive is raised), so I think everything should be OK now.